src/internet_identity/Cargo.toml

```diff
@@ -4,28 +4,32 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-base64 = "0.13.0"
-hex = "0.4"
-ic-cdk = "0.3.2"
-ic-cdk-macros = "0.3"
-ic-certified-map = "0.3.0"
-ic-types = "0.1.1"
-lazy_static = "1.4.0"
-lodepng = "3.6.1"
-serde = "1"
-serde_bytes = "0.11"
-serde_cbor = "0.11"
-serde_with = "1.6.2"
-sha2 = "0.9.1"
-rand_core = "0.5.1"
-rand_chacha = "0.2.2"
-#  We need a custom captcha that allows using custom RNGs
-#  https://github.com/dfinity/internet-identity/issues/472
-captcha = { git = 'https://github.com/nmattia/captcha', rev = '4751b6fa4e56229c2af5b9a28195ff039551d8e7' }
+hex = "*"
```

Comment:
We could use "= 1.2.3" syntax for constraints, I believe this will make reviewing the Cargo.lock PR updates much easier: it will be clear which change in Cargo.toml affected the Cargo.lock.

Reply:
Done!